pkg/kubewarden/detail/policies.kubewarden.io.policyserver.vue

```diff
@@ -7,13 +7,15 @@ import { allHash } from '@shell/utils/promise';
 import CreateEditView from '@shell/mixins/create-edit-view';
 import { mapPref, GROUP_RESOURCES } from '@shell/store/prefs';
 
+import { BadgeState } from '@components/BadgeState';
```

Comment:
curiously, I don't see BadgeState being added to the html part of this PR change... but it's there looking at the whole file. Was it forgotten? 😛

Reply:
Oh yeah, that might be a leftover